blog/2025-03-20-march-releases.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: March 2025 Releases
+slug: 2025-03-20-march-releases
+authors: [kenodegard]
+tags: [announcement, conda, conda-build]
+description: |
+  conda 25.3.0, conda-build 25.3.0, and conda-libmamba-solver 25.3.0 have been released! 🎉
+image: img/blog/2025-03-20-march-releases/banner.png
+---
+
+The March 2025 releases included updates to conda, conda-build, and conda-libmamba-solver! 🎉 All of these have been released to both `main` and `conda-forge` channels.
+
+<!-- truncate -->
+
+## Changes in conda [25.3.0](https://github.com/conda/conda/releases/tag/25.3.0)
+
+To update `conda` to the latest version, run:
+
+```bash
+conda install -n base conda=25.3.0
+```
+
+**Notable Changes:**
+- Add support for Python 3.13.
+- Refactor CMD's activation process to use a static shell script instead of a dynamic temporary script. This resolves using `conda activate` with script execution restrictions when Windows AppLocker is enabled.
```

Comment:
I'm not totally clear on this second sentence. This resolves the need to use `conda activate` with script execution restrictions or does it resolve an issue with using `conda activate`?

Reply:
this resolves an issue where `conda activate` will fail in CMD when AppLocker's script execution restrictions are enabled

Reply:
Ah okay, then my suggestion is to state that more clearly:

```suggestion
- Refactor CMD's activation process to use a static shell script instead of a dynamic temporary script. This resolves an issue where using `conda activate` would fail when Windows AppLocker script execution restrictions were enabled.
```